styles/Datadog/headings.yml

```diff
@@ -140,25 +209,41 @@ exceptions:
   - LaunchDarkly
   - Linux
   - Live Analytics
+  - Live Debugger
+  - Live Messages
+  - Live Processes
   - Live Search
   - Live Tail
+  - LLM Experiments
+  - LLM Observability
   - Log Explorer
   - Log Management
+  - Log Pipeline Scanner
   - Log Rehydration
+  - Log Workspaces
+  - Logging without Limits
   - Logback
   - macOS
   - Marketplace
   - MarkLogic
+  - MCP Server
   - Meraki
   - Mesos Slave Integration
   - Metrics Explorer
   - Metrics without Limits
   - Mobile Monitoring
+  - Mobile App Testing
+  - Mobile Real User Monitoring
+  - Mobile Session Replay
   - MongoDB
   - MsTest
+  - MITRE ATT&CK® Map
   - MySQL
+  - Native Actions
+  - Natural Language Queries
   - Network Address Translation
   - Network Device Monitoring
+  - Network Path
   - Network Performance Monitoring
```

Comment:
```suggestion
```
removing this one, replaced by Cloud Network Monitoring, which you do have already.